cypress/support/fragments/bulk-edit/bulk-edit-search-pane.js

```diff
@@ -884,7 +885,9 @@ export default {
   isDragAndDropAreaDisabled(isDisabled) {
     cy.expect(fileButton.has({ disabled: isDisabled }));
   },
-
+  isCloseAndSaveButtonDisabled(isDisabled) {
```

Comment:
is it closeAndSave button or saveAndClose button? make sure method names are relevant

Reply:
Done!